__tests__/oneTrustCookieConsent.test.js

```diff
@@ -4,12 +4,18 @@ window.OneTrust = {
   GetDomainData: jest.fn(() => ({
     Groups: [
       { CustomGroupId: 'C0001', GroupName: 'Functional Cookies' },
+      { CustomGroupId: 'C0002', GroupName: 'Performance Cookies' },
```

Comment:
can we also add a test in sanity sute for this?

- add a function in trigger that sets dummy concent ids in window for a test case in track
- clear the value afterwards adding a new resetWindow method in the testbook.js that is called after result diff check